### ISSUE TITLE
the loss is none

### ISSUE THREAD

**Issue by xinyuxiao** [xuduo35/MakeLongVideo#6]
I add your weight file 5200 to finetune on webvid, but after 20 steps around, the loss is none, i can not know what is the reason of this problem. thank you!

**Comment by xuduo35** [comment]
What's your hardware setting? Can you tell me your config file? Any training log?

**Comment by xinyuxiao** [comment]
thank you, i have solve this problem, but in finetune, the validation prompts generated gifs lack good consistency. Is this normal？

**Comment by xuduo35** [comment]
Seems infer.py can get better results than validation stage in my experiment. Training long enough will get good enough results.

**Comment by xinyuxiao** [comment]
this is my validation example, after 200 step from 5200, can you judge these gifs @xuduo35 , thank you
![sample-5400](https://github.com/xuduo35/MakeLongVideo/assets/19719974/fa727855-1bfa-491b-9aa3-93148c05a4d4)

**Comment by xuduo35** [comment]
Not bad. Similar result with mine for validation stage. You can try other prompts, refer to https://imagen.research.google/video. My test prompts, they can generate more good videos:
"a glass bead falling into water with huge splash"
"a view of fireworks in Paris "
"a video of Earth rotating in space"
"a cat eating food of a bowl, in von Gogh style"
"Aerial view of a snow-covered mountain. "
"drone flythrough interior of sagrada familia cathedral" 
"A happy elephant wearing a birthday hat walking under the sea."
"cat in the rain"

![a glass bead fal-T7K6fu](https://github.com/xuduo35/MakeLongVideo/assets/6389449/c1fb2925-c43c-4b23-a994-822b8782ad84)
![Aerial view of a-xuQTgc](https://github.com/xuduo35/MakeLongVideo/assets/6389449/e91c2e62-2ac3-4034-91ba-096256e648eb)
![cat in the rain-WWDmyY](https://github.com/xuduo35/MakeLongVideo/assets/6389449/a61a94a3-68a9-4dd9-9c80-95722ac6cd50)
![a video of Earth-3sHbww](https://github.com/xuduo35/MakeLongVideo/assets/6389449/0ca4d691-9b4f-4601-ad83-929c0e0ad4fe)
![a view of firewo-un5L2X](https://github.com/xuduo35/MakeLongVideo/assets/6389449/b66d9352-defb-4329-9826-2d52e80619a5)

![A happy elephant-dgeCVr](https://github.com/xuduo35/MakeLongVideo/assets/6389449/157b4d9e-9389-49bb-8900-40ce5db8eebc)
![Aerial view of a-g1ywKg](https://github.com/xuduo35/MakeLongVideo/assets/6389449/7ba76d11-38ed-44bd-9263-4bb5fdefe2e9)
![drone flythrough-P6gegX](https://github.com/xuduo35/MakeLongVideo/assets/6389449/a3bd7825-361c-4947-86f4-fbb1b41b51b3)
![a view of firewo-FoG0M2](https://github.com/xuduo35/MakeLongVideo/assets/6389449/56b9bf27-5eac-4b24-9a7e-e3490ee6b2c1)
![a video of Earth-NSUtIs](https://github.com/xuduo35/MakeLongVideo/assets/6389449/e241fe29-48e5-4425-adef-2a80e543dced)
![a glass bead fal-KiZCWe](https://github.com/xuduo35/MakeLongVideo/assets/6389449/b6f3cef3-bcb9-420b-aedc-8f8673d8b1a9)
![cat in the rain-XdG4BT](https://github.com/xuduo35/MakeLongVideo/assets/6389449/290b359b-b2f8-4785-ad86-e8454f65fe8d)

**Comment by xinyuxiao** [comment]
OK, thanks. You videos haven't watermark, do train it use new without watermark datasets, is it convenient to provide data sources

**Comment by xuduo35** [comment]
I collect some watermark-free videos, training together with webvid dataset.
